### PR TITLE
Scaffold rails app 

### PR DESCRIPTION
**Pull request by Siianko**
PT stories:
RoR scaffold https://www.pivotaltracker.com/story/show/159757814 

### User story ###
```
As development team,
In order to start working,
We would like to scaffold a RoR app.
```

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# gem 'capistrano-rails', group: :development
+
+# Reduces boot times through caching; required in config/boot.rb
+gem 'bootsnap', '>= 1.1.0', require: false
+
+group :development, :test do
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
```
Comment (faraznaeem) [inline Gemfile]: clean up from comments

**Review by tochman — CHANGES_REQUESTED** [review]
I would clear the Gemfile from comments and unused gems before merge

**Review by faraznaeem — CHANGES_REQUESTED** [review]
Please also set up Haml in a separate chore

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.4.3'
+
+gem 'rails', '~> 5.2.1'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```
Comment (tochman) [inline Gemfile]: Please remove

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
```
Comment (tochman) [inline Gemfile]: Please remove

**Review by ESteinman — COMMENTED** [review]
LGTM

**Review by tochman — APPROVED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**